articles/configure-custom-routing.mdx

```diff
@@ -0,0 +1,190 @@
+</comp.Notice>
+
+To create a blank custom routing file:
+
+1. Navigate to the `.cloudcannon` folder using the *File browser*, or create one in the root of your Site.
+2. Click the *\+ Add* button at the top right of the browser and select the *\+ Create new file* option from the dropdown. CloudCannon will open a blank `.txt` file in the Source Editor.
+3. Click the *Save* button in the top right of the editing interface. CloudCannon will open the *Review changes* modal.
+4. Edit the file path to "/.cloudcannon/routing.json" and click the *Save selected* button in the bottom right of the modal.
+
+CloudCannon will save a blank `routing.json` file in your `.cloudcannon` folder.
+
+If you would prefer, you can also create a `routing.json` file using your local development environment.
+
+For a complete list of options available in the `routing.json` file, please read our [routing reference](/documentation/articles/routing-reference/) documentation.
+
+## Custom Redirects
+
+Custom redirects control what content is delivered to a visitor when they attempt to visit a URL. Most redirects need three parts: the `from` URL, the `to` URL, and the `status` code.
+
+* **From** — Specifies which URL should trigger this redirect rule.
+* **To** — Specifies which URL the visitor should be redirected to.
+* **Status** — Tells the visitor's browser the reason for the redirect.
+
+CloudCannon allows the following [HTTP status codes](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes):
+
+* **200** — Indicates the request was successful.
+* **301** — Indicates the requested URL has been changed permanently and specifies the new URL in the response.
+* **302** — Indicates the requested URL has been changed temporarily, and the file server should continue to use the original URL in future requests.
+* **307** — Indicates the requested URL has been changed temporarily and the file server should continue to use the original URL in future requests. This is similar to a 302 code however it does not allow the HTTP method to change.
+* **308** — Indicates the requested URL has been changed permanently and the file server should continue to use the original URL in future requests. This is similar to a 301 code however it does not allow the HTTP method to change.
+* **404** — Indicates the requested URL could not be found.
+* **410** — Indicates the requested URL has been permanently deleted, and search engines should de-index this page.
+
+Most users will only need to specify redirects using the 301, 302, and 404 status codes. We recommend redirecting your old page URLs with 301 or 302 redirects whenever you change the URL of a page or the architecture of your Site in a way that affects the URL structure. This helps to preserve your SEO scores if you want to rank highly in search engine results, maintains external links to your website, and prevents visitors from seeing incorrect or missing content when they attempt to load any of your old URLs.
```

Comment:
```suggestion
Most users will only need to specify redirects using the 307, 307, and 404 status codes. We recommend redirecting your old page URLs with 307 or 308 redirects whenever you change the URL of a page or the architecture of your Site in a way that affects the URL structure. This helps to preserve your SEO scores if you want to rank highly in search engine results, maintains external links to your website, and prevents visitors from seeing incorrect or missing content when they attempt to load any of your old URLs.
```

307 and 308 are the new "ideal" redirect codes. It doesn't matter much for sites on CloudCannon but best practice is to use them over 301 or 302.